task2/app.js

```diff
@@ -0,0 +1,127 @@
+function createUser() {
+  window.location.href = "./createUser.html";
+}
+function logout() {
+  window.location.href = "./index.html";
+  localStorage.clear();
+}
+
+const container = document.getElementById("container");
+const url = "https://reqres.in/api/users";
+let usersloggedin = new Array();
```

Comment:
We can simply create a new array using array literal instead of using the Array() constructor. Using an array literal provides better readability and better compilation speed.

`let usersloggedin = [ ];`

@shubhamshukla91